### PR TITLE
DL-7694: introduced frontend assets&updated multipleEmplpoymentsView

### PR DESCRIPTION
**Pull request by selinahussain**

### CODE REVIEW

File: app/views/Layout.scala.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+@*
+* Copyright 2021 HM Revenue & Customs
```
Comment (russellthorn) [inline app/views/Layout.scala.html:2]: 2021?

File: app/views/playComponents/answer_row.scala.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
```
Comment (russellthorn) [inline app/views/playComponents/answer_row.scala.html:2]: 2021?

File: app/views/playComponents/answer_row.scala.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+@rowAnswer = @{
+    if(row.answerIsMessageKey) {
+        messages(row.answer).capitalize
+    } else {
+        Html(row.answer)
+    }
+}
+
+<li>
+    <div class="cya-question">@Html(messages(row.label, row.labelArgs: _*))</div>
```
Comment (russellthorn) [inline app/views/playComponents/answer_row.scala.html:34]: this doesn't seem like the current implementation of check your answers https://design-system.service.gov.uk/patterns/check-answers/

Happy to discuss this in more detail if I have the wrong end of the stick

File: app/views/playComponents/answer_row.scala.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <div class="cya-answer">@rowAnswer</div>
+    <div class="cya-change">
+        @if(row.changeUrl) {
+        <a href='@row.changeUrl'>
+            @if(row.editText) {
+                <span aria-hidden="true"> @messages(row.editText.get)</span>
+                @if(row.hiddenText){
+                    <span class="visually-hidden">@messages(row.hiddenText.get)</span>
```
Comment (russellthorn) [inline app/views/playComponents/answer_row.scala.html:42]: again, looks like old classes for visually hidden

File: app/views/playComponents/answer_section.scala.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
```
Comment (russellthorn) [inline app/views/playComponents/answer_section.scala.html:2]: year again

File: app/views/playComponents/answer_section.scala.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+@import viewmodels.AnswerSection
+@import views.html._
+
+@(answerSection: AnswerSection)(implicit messages: Messages)
+
+@if(answerSection.headingKey.isDefined){
+    <h2>@messages(answerSection.headingKey.get)</h2>
+}
+
+<ul class="govuk-check-your-answers cya-questions-long section">
```
Comment (russellthorn) [inline app/views/playComponents/answer_section.scala.html:26]: we should be using dl's for check your answers pages - https://design-system.service.gov.uk/patterns/check-answers/

If not using hmrc-frontend then I can help dig out code and classes for using dl's with assets-frontend

**Review by danny-waughman — APPROVED** [review]

**Pull request merged**